batch/batch.go

```diff
@@ -23,8 +23,11 @@ import (
 	"time"
 )
 
+// DefaultWaitInterval is the default WaitInterval for Func.
+var DefaultWaitInterval = 500 * time.Microsecond
+
 // DefaultMaxDuration is the default MaxDuration for Func.
-var DefaultMaxDuration = 200 * time.Microsecond
+var DefaultMaxDuration = 10 * time.Millisecond
```

Comment:
Agreed. I am starting with DefaultWaitDuration = 200us, which was the old DefaultMaxDuration. There is no magic constant that will be perfect for all use cases; these values are environment and request specific. The main use case we should be trying to solve with this implementation is a single GraphQL request that fans out to multiple batch requests. From the traces I've analyzed, this does satisfy that use case.